pahfit/base.py

```diff
@@ -512,14 +546,71 @@ def save(self, obs_fit, filename, outform):
                     ]
                 )
 
+        # Call featcombine to calculate combined dust feature strengths.
+        cftable = self.featcombine(line_table)
+
         # stack the tables (handles missing columns between tables)
-        out_table = vstack([bb_table, line_table, att_table])
+        out_table = vstack([bb_table, line_table, att_table, cftable])
 
         # Writing output table
         out_table.write(
             "{}_output.{}".format(filename, outform), format=outform, overwrite=True
         )
 
+    @staticmethod
+    def featcombine(ftable):
+        """
+        Combine dust features strengths.
+
+        Parameters
+        ----------
+        ftable : data table
+            Astropy table containing lines.
+
+        Returns
+        -------
+        cftable : data table
+            Astropy table containing combined dust feature strengths.
+        """
+        # Define combined strength dictionary.
+        cfdic = {'PAH_62': {'range': [6.2, 6.3]},
+                 'PAH_77_C': {'range': [7.3, 7.9]},
+                 'PAH_83': {'range': [8.3, 8.4]},
+                 'PAH_86': {'range': [8.6, 8.7]},
+                 'PAH_112': {'range': [11.2, 11.4]},
```

Comment:
Note that in PAHFIT the combined PAH feature at 11 µm is dubbed "11.3 complex". For consistency, might be better to name it "PAH_113_C". If we decide to stick with "PAH_112_C", will need to mention it somewhere to avoid confusion.